l10n_ch_import_cresus/tests/test_import.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# © 2016 Bettens Louis (Open Net Sarl)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import openerp.tests.common as common
+from base64 import b64encode
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class TestImport(common.TransactionCase):
+
+    def setUp(self):
+        super(TestImport, self).setUp()
+
+        tax_obj = self.env['account.tax']
+        account_obj = self.env['account.account']
+        analytic_account_obj = self.env['account.analytic.account']
+
+        user_type = self.env['account.account.type'].search(
+            [('include_initial_balance', '=', False)], limit=1)
+        for code in '1000 1010 1210 2200 2800 2915 6512 6513 6642 '\
+                    '9100 10101'.split():
```

Comment:
tbh the main reason is that I was too lazy to add a comma manually. It does the same in less space tho, and it echoes Ruby's `%w,1000 1010 1210 2200,` quite nicely IMHO